chcemvediet/templates/main/homepage/login.html

```diff
@@ -52,7 +52,8 @@ <h2 id="{% trans 'main:homepage:login:anchor' %}">
       </div>
     </div>
   </div>
-  <div>
+  <div class="visible-md chv-above-1"></div>
+  <div class="chv-above-1">
```

Comment:
@martinmacko47 Mozem pouzit padding namiesto margin ako si pisal v zadani?

Reply:
Jasne, v zadani som padding/margin nerozlisoval. Implementovat to mozes podla toho, co je vhodnejsie pouzit.